src/charm.py

```diff
@@ -303,7 +318,7 @@ def _log_rotate_layer(self) -> Layer:
                     "command": f"sh -c 'logrotate {Config.LogRotate.RENDERED_TEMPLATE}; sleep 1'",
                     "startup": "enabled",
                     "override": "replace",
-                    "backoff-delay": "1m",
+                    "backoff-delay": "1m0s",
```

Comment:
Yes it is!
When we `get_plan`, the backoff-delay is returned as `1m0s`.
Since I'm comparing the services completely, I sadly have to do this.
I can try to be a bit more accurate in what I compare but I'd rather give a full comparison :)

Reply:
or more why is `1m0s` rather than `1m`

Reply:
This is out of my hands.
It's returned like this by pebble when I `get_plan` for comparison, I think it's "just" an opiniated choice from pebbele.